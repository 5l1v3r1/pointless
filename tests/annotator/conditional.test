
x = if a then b else c
a = false
b = 2
c = 3