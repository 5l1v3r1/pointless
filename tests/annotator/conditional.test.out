
x = if a then b else c
access: a global: 1
access: b global: 2
access: c global: 3
entry: x index: 0

a = false
entry: a index: 1

b = 2
entry: b index: 2

c = 3
entry: c index: 3
